### ISSUE TITLE
Mouse does not control cursor

### ISSUE THREAD

**Issue by mlinchitstest** [syl20bnr/spacemacs#8194]
``<<tried to file from within spacemacs>>`` but <kbd>C-c C-c</kbd> is `undefined.`

For some reason this happens every few days. Maybe an accidental key-press? The mouse no longer selects regions or places the cursor. It can still scroll and select things in the <kbd>M-m</kbd> minibuffer command palette, but it does not control the cursor. You have to restart the program to fix it, which is kind of irritating when you're working with the `.spacemacs` file (I don't want to save it before I'm done, because it will cause errors, and you cannot copy text with mouse). 

The only external packages I have are tabbar and ob-browser. I didn't change any configs that might affect mouse behavior.

If this is caused by a keypress, it is probably undesired behavior for new users (or maybe even most users).

**Comment by mlinchitstest** [comment]
i haven't closed the program yet. anything i can dump to help you reproduce it?

**Comment by d12frosted** [comment]
Sorry, but I don't really understand what exactly you do. Could you please share more clear reproduction steps?

**Comment by mlinchitstest** [comment]
that's the thing. I don't know what is causing this to happen. i just know that right now I cannot select any text or position the cursor with a mouse click. I think the last thing i was doing was `spacemacs-comment-uncomment-region` or some such—but i'd doubt that's relevant.

**Comment by d12frosted** [comment]
@mlinchitstest could you please share the output of `SPC h d s`? 😸

**Comment by mlinchitstest** [comment]
#### System Info :computer:
- OS: windows-nt
- Emacs: 25.1.1
- Spacemacs: 0.200.6
- Spacemacs branch: nil (rev. nil)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(html helm auto-completion better-defaults emacs-lisp git tabbar themes-megapack markdown org bibtex spell-checking
      (spell-checking :variables =enable-flyspell-auto-completion= t)
      syntax-checking)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS

**Comment by nixmaniack** [comment]
Unrelated note: In your layers listing`=enable-flyspell-auto-completion=` should be just `enable-flyspell-auto-completion` i.e. no surrounding `=`.

Related: 
1. Have you checked if going back to vanilla installation works as expected? (i.e. removing `.emacs.d `and `.spacemacs` and then clone spacemacs and start emacs)
2. If `1` fails - what's the behaviour in vanilla emacs(i.e. without spacemacs)?

**Comment by duianto** [comment]
I also ran into the 
>Spacemacs branch: nil (rev. nil) 

issue when i started using spacemacs. Here's the comment where i mentioned and apparently solved the issue: https://github.com/syl20bnr/spacemacs/issues/6868#issuecomment-241194650

I think the problem was that i had downloaded the spacemacs `.zip` file from the `Clone or download` > `Download ZIP` buttons/links on the main spacemacs github page. The `.zip` file doesn't contain a `.git` folder.

When i used the git clone command to get spacemace then `nil` was replaced with the branch name and hash. Probably because a `.git` folder also gets downloaded.
